Unable to run shell command

Hi! I'm using your MM module and loving it so far, thanks for your work!

Unfortunately I recently run into issues with it. All of a sudden I realized it was no longer turning on and off my monitor, and found this related error in the logs:
```
0|magicmir | [2021-09-26 15:54:09.290] [ERROR]  Whoops! There was an uncaught exception...
0|magicmir | [2021-09-26 15:54:09.294] [ERROR]  TypeError: exec is not a function
0|magicmir |     at Class.turnDisplayOff (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_helper.js:85:9)
0|magicmir |     at MqttClient.<anonymous> (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_helper.js:63:26)
0|magicmir |     at MqttClient.emit (events.js:200:13)
0|magicmir |     at MqttClient._handlePublish (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/mqtt/lib/client.js:1162:12)
0|magicmir |     at MqttClient._handlePacket (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/mqtt/lib/client.js:351:12)
0|magicmir |     at work (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/mqtt/lib/client.js:283:12)
0|magicmir |     at Writable.writable._write (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/mqtt/lib/client.js:294:5)
0|magicmir |     at doWrite (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/readable-stream/lib/_stream_writable.js:428:64)
0|magicmir |     at writeOrBuffer (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/readable-stream/lib/_stream_writable.js:417:5)
0|magicmir |     at Writable.write (/home/pi/MagicMirror/modules/mmm-toggle-by-mqtt/node_modules/readable-stream/lib/_stream_writable.js:334:11)
0|magicmir | [2021-09-26 15:54:09.307] [ERROR]  MagicMirror will not quit, but it might be a good idea to check why this happened. Maybe no internet connection?
0|magicmir | [2021-09-26 15:54:09.310] [ERROR]  If you think this really is an issue, please open an issue on GitHub: https://github.com/MichMich/MagicMirror/issues
```

I don't know why it started happening, because I didn't update or change either the `tmp` folder permissions, MM, Node, ShellJS or your module, but I found a related ShellJS issue where they're still trying to find an explanation: https://github.com/shelljs/shelljs/issues/1024

Anyway I managed to fix it by getting rid of ShellJS in favor of Node's native `child_process`, so I'll be opening a PR soon... I guess there's no need to specify all the versions or how to reproduce the error (I wouldn't know how anyway).

--- a/node_helper.js
+++ b/node_helper.js
@@ -8,8 +8,7 @@
 
 var NodeHelper = require('node_helper');
 var mqtt = require('mqtt');
-var shell = require('shelljs');
-shell.config.execPath = '/usr/bin/node';
+var { exec } = require('child_process');
 
 module.exports = NodeHelper.create({
 
@@ -76,12 +75,12 @@ module.exports = NodeHelper.create({
 
     // Turn display (hdmi) on
     turnDisplayOn: function() {
-        shell.exec('vcgencmd display_power 1 >/dev/null 2>&1');
+        exec('vcgencmd display_power 1 >/dev/null 2>&1');
     },
 
     // Turn display (hdmi) off
     turnDisplayOff: function() {
-        shell.exec('vcgencmd display_power 0 >/dev/null 2>&1');
+        exec('vcgencmd display_power 0 >/dev/null 2>&1');
     }
 
 });

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "mmm-toggle-by-mqtt",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Shows/hides MagicMirror modules on mqtt command",
   "main": "mmm-toggle-by-mqtt.js",
   "scripts": {
@@ -14,7 +14,6 @@
   "author": "Moritz Kraus",
   "license": "MIT",
   "dependencies": {
-    "mqtt": "^3.0.0",
-    "shelljs": "^0.8.3"
+    "mqtt": "^3.0.0"
   }
 }

